source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
```

Comment:
Added support for Electron BrowserView, an underlying architecture change that improves performance and offers snappier interactions (i.e., less lag), lower CPU usage, and faster launch times.